docs/setup/install-with-keadm.md

```diff
@@ -2,51 +2,54 @@
 title: Installing KubeEdge with Keadm
 sidebar_position: 3
 ---
-Keadm is used to install the cloud and edge components of KubeEdge. It is not responsible for installing K8s and runtime.
 
-Please refer [kubernetes-compatibility](https://github.com/kubeedge/kubeedge#kubernetes-compatibility) to get **Kubernetes compatibility** and determine what version of Kubernetes would be installed.
+Keadm is used to install the cloud and edge components of KubeEdge. It does not handle the installation of Kubernetes and its runtime environment.
 
-## Limitation
+Please refer to [Kubernetes compatibility](https://github.com/kubeedge/kubeedge#kubernetes-compatibility) documentation to check **Kubernetes compatibility** and ascertain the Kubernetes version to be installed.
 
-- Need super user rights (or root rights) to run.
+## Limitation
 
+- It Requires super user rights (or root rights) to run.
 
 ## Install keadm
 
-There're three ways to download a `keadm` binary
+There're three ways to download the `keadm` binary:
 
-- Download from [github release](https://github.com/kubeedge/kubeedge/releases).
+1. Download from [GitHub release](https://github.com/kubeedge/kubeedge/releases).
   
-    Now KubeEdge github officially holds three arch releases: amd64, arm, arm64. Please download the right arch package according to your platform, with your expected version.
+    KubeEdge GitHub officially holds three architecture releases: amd64, arm, and arm64. Please download the correct package according to your platform and desired version.
+
     ```shell
     wget https://github.com/kubeedge/kubeedge/releases/download/v1.12.1/keadm-v1.12.1-linux-amd64.tar.gz
     tar -zxvf keadm-v1.12.1-linux-amd64.tar.gz
     cp keadm-v1.12.1-linux-amd64/keadm/keadm /usr/local/bin/keadm
```

Comment:
we are not tracking this example version, but v1.12.1 seems pretty old, can we instead refer to v1.16.2 at this time?

```suggestion
    wget https://github.com/kubeedge/kubeedge/releases/download/v1.16.2/keadm-v1.16.2-linux-amd64.tar.gz
    tar -zxvf keadm-v1.16.2-linux-amd64.tar.gz
    cp keadm-v1.16.2-linux-amd64/keadm/keadm /usr/local/bin/keadm
```

Reply:
I agree!